oca_event_badge/static/src/css/event_event_template.scss

```diff
@@ -0,0 +1,8 @@
+/* Copyright 2018 ACSONE SA/NV
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+
+.badge_logo {
+    max-height:1cm;
```

Comment:
Use 'cm' to display in screens are not recommended.
See https://www.w3.org/Style/Examples/007/units.en.html

Reply:
I would gladly change it but, I don't have write permissions already in this repo. Just out of curiosity, what unit of measure would you recommend?

Reply:
@Tardo I would say this is for printing the badge on paper, so it's not on screen.